packages/prettier-config/README.md

```diff
@@ -0,0 +1,60 @@
+# Linting ESLint
+
+> Part of Open Web Component Recommendation [open-wc](https://github.com/open-wc/open-wc/) Recommendations [open-wc](https://open-wc.org/)
+
+We want to provide a good set of defaults on how to facilitate your web component.
+
+[![CircleCI](https://circleci.com/gh/open-wc/open-wc.svg?style=shield)](https://circleci.com/gh/open-wc/open-wc)
+[![BrowserStack Status](https://www.browserstack.com/automate/badge.svg?badge_key=M2UrSFVRang2OWNuZXlWSlhVc3FUVlJtTDkxMnp6eGFDb2pNakl4bGxnbz0tLUE5RjhCU0NUT1ZWa0NuQ3MySFFWWnc9PQ==--86f7fac07cdbd01dd2b26ae84dc6c8ca49e45b50)](https://www.browserstack.com/automate/public-build/M2UrSFVRang2OWNuZXlWSlhVc3FUVlJtTDkxMnp6eGFDb2pNakl4bGxnbz0tLUE5RjhCU0NUT1ZWa0NuQ3MySFFWWnc9PQ==--86f7fac07cdbd01dd2b26ae84dc6c8ca49e45b50)
+[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
+
+Uses [Prettier](https://prettier.io) to format your JS, CSS and HTML code.
+
+## Manual
+
+- Install `@open-wc/prettier-config`.
+
+```bash
+yarn add @open-wc/prettier-config --dev
+```
+
+- Create `.prettierrc.js` to root directory of the project as following.
+
+```js
+module.exports = require('@open-wc/prettier-config');
+```
+
+- Add these lines to your package.json
+
+```js
+"husky": {
```

Comment:
I think I would prefer for both husky and lint-staged the `*.config.js` version
e.g. `lint-staged.config.js` and `husky.config.js`

what do you think?

according to http://wiki.commonjs.org/wiki/Packages/1.1
> Package managers and loaders should ignore unknown fields in the package descriptor file.

so it's sort of not in line (they have more Reserved Properties - rest "could be" open) but it's also very common...

Reply:
Good point, thanks.